tsconfig.json

```diff
@@ -19,6 +19,7 @@
       "@utils/*": ["utils/*"],
       "@config/*": ["config/*"],
       "@assets/*": ["assets/*"],
+      "@components/checkout/*": ["framework/shopify/components/checkout/*"],
```

Comment:
I don't think this is required, the way we currently import hooks should also work for components.